### PR TITLE
Concourse 3.7.0

### PR DESCRIPTION
**Pull request by thomasmmitchell**
* Bumps to Concourse 3.7.0
* Uses Concourse's own TLS termination instead of at the HAProxy level to avoid confusing Fly (and by extension, confusing the operator) when you try to talk to Concourse with plebeian HTTP unencrypted slop.
* The kit now asks how many web nodes you want. We also now support having up to 5 web nodes.

### CODE REVIEW

File: kit.yml [hunk 1]
```diff
@@ -36,6 +36,11 @@ params:
         Do not include 'https://' in this value.
       param: external_domain
 
+    - ask: How many web frontend nodes would you like to deploy and load balance between?
+      description: The number of instances of web nodes to allocate.
```
Comment (jhunt) [inline kit.yml]: Can we add a note to the ask/description that the highest allowed value is 5?

File: kit.yml [hunk 2]
```diff
@@ -165,6 +170,8 @@ credentials:
     tsa/host_key: ssh 2048 fixed
     tsa/worker_key: ssh 2048 fixed
 
+    atc/token_signing_key: rsa 2048 fixed
```
Comment (jhunt) [inline kit.yml]: Just to check my assumption, the ATC Token Signing Key is fixed because it signs long-lived tokens that are outside the scope of the deployment (the fly cli) and rotating it regularly would cause a bad end-user experience? (_why do I keep getting logged out?!_)

Comment (thomasmmitchell) [reply kit.yml]: That was the idea, yeah. Whether or not this should be "fixed" is questionable, because its not so bad if its a once a month cycle, but daily or weekly cycles of this could be frustrating. I chose to err on the safe side.

I'm not sure why host and worker keys are fixed though. I mean, if you have external deployments of the workers... then, sure, worker key. Host key is between atc and tsa, so... is should that really be fixed?

File: kit.yml (hunk 1 above)
Comment (thomasmmitchell) [reply kit.yml]: Yeah - I wasn't sure if the user was going to get any prompt about the validation range.

File: kit.yml (hunk 2 above)
Comment (thomasmmitchell) [reply kit.yml]: I take it back. Its for the database. It would need to be fixed if its an external deployment or provided by your IaaS
